tasks/main.yml

```diff
@@ -2,21 +2,33 @@
 
 # @see http://stackoverflow.com/questions/19529999/add-package-control-in-sublime-text-3-through-the-command-line
 
-- name: Sublime Text 3 | Package Control | Download
-  get_url: 
-    url: "{{ packagecontrol_url }}"
-    dest: "~/.config/sublime-text-3/Installed Packages"
-
-- name: Sublime Text 3 | Package Control | Set owner and rights
+- name: "Sublime Text 3 | Package Control | Ensure package dirs owner and rights"
+  sudo: Yes
   file:
-    state: file
-    path: "~/.config/sublime-text-3/Installed Packages/Package Control.sublime-package"
-    mode: 0666
+    state: directory
+    path: "{{ item }}"
+    mode: "u=rw,g=rw,o=rw"
     owner: "{{ packagecontrol_owner }}"
     group: "{{ packagecontrol_group }}"
+  with_items:
+    - "{{ packagecontrol_dir_bin | expanduser }}"
```

Comment:
I think this must be discussed a little bit.

I noticed when tested role that `expanuser` expands only local user. Running this role on remote machine (e.g. vagrant box) have unexpected (as for me) behavior.

I asked about this here: https://github.com/ansible/ansible/issues/10670#issuecomment-92358611 and realize that this is not a bug.

On the one hand, this role seems to be used only locally (for me). On the other hand, that breaks tests on vagrant box or make unable to install that role on remote machine.

WDYH?